Demo/Demo/Features/FeatureBaseViewController.swift

```diff
@@ -0,0 +1,91 @@
+import UIKit
+
+class FeatureBaseViewController: UIViewController {
+
+    let containerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
```

Comment:
This is neat! Really cool way of consolidating all view customization logic for this property in one place.